src/components/virtualRepeat/virtualRepeater.js

```diff
@@ -22,7 +22,9 @@ angular.module('material.components.virtualRepeat', [
  *
  * @usage
  * <hljs lang="html">
- * <md-virtual-repeat-container></md-virtual-repeat-container>
+ * <md-virtual-repeat-container>
+ *   <div md-virtual-repeat="i in items" md-item-size="10">Hello {{i}}!</div>
+ * </md-virtual-repeat-container>
```

Comment:
@gkalpak - What are your thoughts here? It sees that md-item-size is misleading; 
*  the attribute is not the size but rather the md-num-visible-items, and  
*  the actual pixel size is dependent upon the full-size of the md-virtual-repeat-container viewport.